src/rpc/rpc.js

```diff
@@ -144,6 +132,7 @@ bloombox.rpc.ScopedOptions = (class ScopedOptions {
      * Partnership scope to use as an override for this RPC, if applicable.
      * Defaults to null if left unset.
      *
+     * @const
      * @protected
```

Comment:
it noticed that lots of properties can be labeled `@const`, which i did not know.